bulletroboy/exoforce/exoforce_hardware.py

```diff
@@ -0,0 +1,266 @@
+import numpy as np
+from pyquaternion import Quaternion
+
+from .exoforce import ExoForce
+from ..utils.utils import Topics, Services
+
+from roboy_simulation_msgs.msg import Collision
+from roboy_middleware_msgs.msg import MotorCommand, MotorState
+from roboy_middleware_msgs.srv import ControlMode
+from geometry_msgs.msg import PoseStamped
+
+from rclpy.callback_groups import ReentrantCallbackGroup
+
+FAST_PWM_TENSION = 300
+MIN_PWM_TENSION = 150
```

Comment:
must be configured as parameters and not hardcoded here

Reply:
This was a fast configuration for last semester, when the force control is done we wont use PWM values at all in the exoforce node

Reply:
see https://github.com/Roboy/bulletroboy/pull/25#issuecomment-760128816